bedrock/src/backup/mod.rs

```diff
@@ -328,6 +336,15 @@ impl BackupManager {
         crate::info!("Cleaning up backup system... Deleting manifest file after backup is disabled/deleted.");
         let manifest = ManifestManager::new();
         manifest.danger_delete_manifest()?;
+
+        // After deleting the manifest, refresh the base report so it no longer reports
+        // any backup designators or size for a backup that has been disabled.
+        if let Err(e) = ClientEventsReporter::new().sync_base_report_with_manifest() {
```

Comment:
actually wouldn't it be better to just delete the base report in this case? it should be re-created if/when the user enables the backup again